packages/client/test/service/lightethereumservice.spec.ts

```diff
@@ -12,12 +12,12 @@ tape('[LightEthereumService]', async (t) => {
   }
   PeerPool.prototype.open = td.func<any>()
   PeerPool.prototype.close = td.func<any>()
-  td.replace('../../lib/net/peerpool', { PeerPool })
+  td.replace<any>('../../lib/net/peerpool', { PeerPool })
   const MockChain = td.constructor([] as any)
   MockChain.prototype.open = td.func()
-  td.replace('../../lib/blockchain', { MockChain })
+  td.replace<any>('../../lib/blockchain', { MockChain })
   const LesProtocol = td.constructor([] as any)
-  td.replace('../../lib/net/protocol/lesprotocol', { LesProtocol })
+  td.replace<any>('../../lib/net/protocol/lesprotocol', { LesProtocol })
```

Comment:
just wondering why do we have to now explicitly type it to any?

Reply:
Testdouble did a minor version upgrade that got picked up when I added it to `tx` and `block` as a devDependency and their `td.replace` function now explicitly returns an object of type `unknown` instead `any` if you don't specify the type to return.  I can back this version bump out and pin to the previous version if people are concerned about it.